CONTRIBUTING.md

```diff
@@ -0,0 +1,185 @@
+
+# Getting Started
+
+## Non-code contributions
+
+There are several ways to help out without coding, these are primarily:
+
+- Opening issues for bugs.
+- Opening issues for data problems.
+- Opening and/or participating in discussions about new data sources, features, and more.
+- Opening issues when capacity data sources have been updated or changed.
+- Finding new data sources, wiki page: [Find data sources][wiki find data sources]
+- Verify existing data sources, wiki page: [Verify data sources][wiki verify data sources]
+- Adding new or updating our existing translations, wiki page: [Translating app.electricitymaps.com][wiki translating the app]
+- And more!
+
+> **Note**
+> Take a look at the wiki pages for more detailed instructions
+
+## Zone and exchange configurations
+
+Static information about zones and exchanges are located at [config/zones][config zones] and [config/exchanges][config exchanges] respectively.
+The zone configurations hold information such as the installed capacity, which parsers to use, fallback mixes, contributors and other keys that are required by our frontend and internal systems; and while similar the exchange configs hold the the location, capacity, direction and parsers required.
+
+## Parser guidelines
+
+To get stated with editing the parsers use the following steps:
+
+1. Run `poetry install -E parsers` to install all needed dependencies.
+2. Use `poetry run test_parser ZONE_KEY` to test any parser changes.
+
+Note: This requires you to have [Python][python homepage] and [Poetry][poetry homepage] installed, you can see their respective installation guides here:
+
+- [Downloading Python][python install guide]
+- [poetry installation][poetry install guide]
+
+### Parser requirements
+
+There are no clear cut model or template that works for all parsers as all data are different and unique but there are some basic requirements all parsers need to follow:
+
+- They need to return valid data from a credible source.
+- They need to return the date for when the data was collected together with the data.
+- They need to return at least one valid Electricity Maps data object (production, consumption, exchange, price, etc).
+  Example of a production return object:
+
+```python
+{'datetime': datetime.datetime(2023, 1, 16, 15, 0, tzinfo=tzutc()),
+  'production': {'biomass': 542.0,
+                 'coal': 192.0,
+                 'gas': 142.0,
+                 'geothermal': None,
+                 'hydro': None,
+                 'nuclear': None,
+                 'oil': 34.0,
+                 'solar': 0.0,
+                 'unknown': None,
+                 'wind': 1164.0},
+  'source': 'entsoe.eu',
+  'storage': {'hydro': None},
+  'zoneKey': 'DK-DK2'}
+```
+
+Note: If data from a production mode is missing it should be omitted or returned as `None` like the above example, _NOT_ `0`.
+
+<!-- TODO: Create wiki pages for all types of return types and provide more examples then link them here. -->
```

Comment:
This can be replaced by a link to the examples folder now

Reply:
The examples folder only have one example + the example parser so far. Wouldn't we want to wait until we have at least a few more examples in there?

Reply:
Let's replace it and I'll see if I can add the other examples this week.